_posts/2016-12-x-kinetic-release.markdown

```diff
@@ -0,0 +1,44 @@
+---
+author: 130s
+comments: false
+date: 2016-12-x 00:00:00+00:00
+layout: post
+slug: firstkineticrelease
+title: MoveIt! officially released into ROS Kinetic
+media_type: image
+media_link: http://wiki.ros.org/kinetic?action=AttachFile&do=get&target=kinetic.png
+description: If you're running your robot packages with ROS Kinetic, and/or if you want to try the newest MoveIt!, start at moveit.ros.org/install
+categories:
+- MoveIt!
+- ROS
+---
+
+MoveIt! finally made it to the latest ROS LTS (Long Term Support) version [Kinetic](http://wiki.ros.org/kinetic)!
```

Comment:
Sounds negative - how about "We are happy to announce MoveIt! is now released into ROS Kinetic LTS (Long Term Support)"